griffon/services/core_queries.py

```diff
@@ -339,9 +339,7 @@ def execute(self, status=None) -> List[Dict[str, Any]]:
             "include_fields": "purl,type,name,related_url,namespace,software_build,nvr,release,version,arch,product_streams.product_versions,product_streams.name,product_streams.ofuri,product_streams.active,product_streams.exclude_components,product_streams.relations",  # noqa
         }
 
-        component_name = (
-            re.escape(self.component_name) if not self.regex_name_search else self.component_name
-        )
+        component_name = self.component_name
```

Comment:
This is a regression. In GRIF-177 -r option was added so we can alter between regex and non regex search. If we don't escape here, then commands which does not include -r and has some potential regex special characters might throw errors.

Reply:
as discussed the logic is a little more involved eg. we need to test if -r or -s is being used ... have amended